### PR TITLE
Drg/feature/audd.io ajax call

### LINKED ISSUE #102: Create Audd.io ajax call
Put together a template ajax call that will return the lyrics of a song.

This is so that we can then copy this across the different functions it will be used and change where queryURL is being pulled from

### PR DESCRIPTION
**Pull request by drg0drg**

### CODE REVIEW

File: assets/script.js [hunk 1]
```diff
@@ -72,6 +72,28 @@ function getTrackInfo(userInput) {
       //show the result page after finish call
       $("#track-results-page").show(400);
     }
+    //"reading" the value of <li>
+    // $('#tracks').each(function() {
+      // $('#track-search-result>ol>li').on('click', function() {
```
Comment (Goldenaries658) [inline assets/script.js]: Are you going to use this, if not remove old comments and clean up code.

File: assets/script.js [hunk 2]
```diff
@@ -72,6 +72,28 @@ function getTrackInfo(userInput) {
...
+        $('.track-result').on('click', function() {
+          var getLyricsFor = $(this).text();
```
Comment (Goldenaries658) [inline assets/script.js]: Change this variable name to be consistent with the rest of the script e.g. track

File: assets/script.js [hunk 3]
```diff
@@ -72,6 +72,28 @@ function getTrackInfo(userInput) {
...
+          console.log("clicked is: " + getLyricsFor);
```
Comment (Goldenaries658) [inline assets/script.js]: Clean up code

File: assets/script.js [hunk 4]
```diff
@@ -72,6 +72,28 @@ function getTrackInfo(userInput) {
...
+          //Create lyrics api url with <li> information. The syntax is //https://api.audd.io/findLyrics/?q=adele hello
+          var lyricsURL = "https://api.audd.io/findLyrics/?q=" + getLyricsFor + "&api_token=56504f66202634311b0e5c04f32ced06"  
+          console.log(lyricsURL);
```
Comment (Goldenaries658) [inline assets/script.js]: Clean up code

File: assets/script.js [hunk 5]
```diff
@@ -72,6 +72,28 @@ function getTrackInfo(userInput) {
...
+          $.ajax({
+            url: lyricsURL,
+            method: "GET", 
+          }).then(function(response){
+            console.log(response);
```
Comment (Goldenaries658) [inline assets/script.js]: Clean up code

File: assets/script.js [hunk 6]
```diff
@@ -72,6 +72,28 @@ function getTrackInfo(userInput) {
...
+            var responseLyrics = response.result[0].lyrics;
+            console.log("lyrics for button: " + responseLyrics);
+            $("#modal-track-search-result").text(responseLyrics);
+          })
+          });
+      
+      // })
+
```
Comment (Goldenaries658) [inline assets/script.js]: What's going on here?

**Review by Goldenaries658 — CHANGES_REQUESTED** [review]
Work out what it is that you are doing that keeps creating submodules.

File: assets/script.js (hunk 1 above)
Comment (drg0drg) [reply assets/script.js]: done in latest commit

File: index.html [hunk 7]
```diff
@@ -174,7 +174,7 @@ <h2 id="track-name"></h2>
         <h3>Track Search Results</h3>
         <div class="search-tracks" id="track-search-result">
           <ol>
-            <li></li>
+            <!-- <li></li> -->
```
Comment (Goldenaries658) [inline index.html]:
```suggestion
```

**Review by Goldenaries658 — CHANGES_REQUESTED** [review]

File: index.html [hunk 8]
```diff
@@ -222,7 +222,7 @@ <h3>Album Tracks</h3>
             >close</a
           >
         </div>
-        <h1>About Track</h1>
+        <h1></h1>
```
Comment (Goldenaries658) [inline index.html]: Add ID

**Review by Goldenaries658 — CHANGES_REQUESTED** [review]

File: index.html (hunk 7 above)
Comment (drg0drg) [reply index.html]: deleted the <li> and added id on <h1> modal in track result page

**Review by Goldenaries658 — APPROVED** [review]

**Pull request merged**